recipes/sirius-ms/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "sirius" %}
+{% set version = "4.9.8" %}
+
+package:
+  name: {{ name|lower }}-ms
+  version: {{ version }}
+
+source:
+  - url: https://github.com/boecker-lab/sirius/releases/download/v{{ version }}/{{ name }}-{{ version }}-linux64.zip  # [linux]
+    sha256: 0312915249181df975a23b605608b69badd4a657236a8bb7a0aa174a840ddc56  # [linux]
+  - url: https://github.com/boecker-lab/sirius/releases/download/v{{ version }}/{{ name }}-{{ version }}-osx64.zip  # [osx]
+    sha256: ed9e853974feb942b6833c5ce0685c9307f074bc6519a4428c45c11859ae508f  # [osx]
+  - url: https://github.com/boecker-lab/sirius/releases/download/v{{ version }}/{{ name }}-{{ version }}-win64.zip  # [win]
+    sha256: e4a4b83e361ad50a2562acbe4b2aa0593f1d032b2bb6696a8163ef76ba837bff  # [win]
+
+build:
+  number: 0
+
+requirements:
```

Comment:
I'm not very familiar with java but it seems like the JRE is being included in this package? Shouldn't this be depending on `openjdk` instead?

Reply:
The package does no longer contains a bundled jre but uses  https://github.com/conda-forge/openjdk-feedstock instead.